nasa-gibs/worldview
# PWA: Theme Color, Apple Touch Icon, Maskable Icon

## Description

This PR is to confirm that the following work in the PWA.  
- [ ] Theme color is `#212529` which a dark, cobalt blue.
- [ ] The apple touch icon is the NASA meatball with the cobalt blue background.
- [ ] The maskable icon is a more "zoomed" out version of the NASA meatball on a cobalt blue background.  The extra space allows a given OS to mask the icon (square, rounded square, circle, etc.) without obscuring the logo.

[If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...]

## How To Test

1. Pull down this branch and run npm run build && npm start
2. Open [localhost](http://localhost:3000/)
3. Open Chrome Dev tools by pressing F12 or CTRL/CMD+SHIFT+i
4. Click the "Lighthouse" tab and select only "Progressive Web App" on the checkboxes.  You can select either "Mobile" or "Desktop" for the radio buttons.
5. Click "Generate Report".
6. All results should come back green.
7. You may also install the PWA via either the install icon in the URL bar or from the settings menu.
